promscale/installation/kubernetes/tobs.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Install the observability stack (tobs) for Kubernetes
+excerpt: Install tobs
+product: promscale
+keywords: [analytics, Kubernetes]
+tags: [install, k8s]
+related_pages:
+  - /promscale/:currentVersion:/installation/resource-recomm
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleConfigurationRecommendations from 'versionContent/_partials/_promscale-configuration-recommendations.mdx';
+
+# Install the observability stack (tobs) for Kubernetes
+The Observability Stack, is a pre-packaged distribution of observability tools
+and dashboard interfaces which can be installed on any existing Kubernetes
+cluster. It includes many of the most popular open-source observability tools
+such as Prometheus, Grafana, Jaeager, Promlens, TimescaleDB, and others.
```

Comment:
```suggestion
including Prometheus, Grafana, Jaeager, Promlens, and TimescaleDB.
```

Reply:
Is there a complete list in a canonical place somewhere? Seem like it would be good to have a line here about "For a complete list of tools included in tobs, see [LINK]".